### ISSUE TITLE
NIST / PCI-DSS compliance integration

### ISSUE THREAD

**Issue by FlorentATo** [mozilla/http-observatory-website#61]
https://www.htbridge.com/ssl/ provides a public API and detailed compatibility with _NIST Special Publication 800-52 Revision 1 - Section 3_ and _PCI DSS 3.2 - Requirements 2.3 and 4.1_ (+ some best-practices as bonus).

I might be useful to integrate those metrics and perhaps subtly remind the users that, as of today, complying with NIST recommendations doesn't necessarily mean "being secure".

**Comment by marumari** [comment]
Yep, and at one point we had integrated with HT Bridge.  I'm waiting for them to stabilize and version their API before I start using it again.  :)

**Comment by GaryGapinski** [comment]
I've attempted to get in touch with HT Bridge regarding NIST compliance. Haven't heard back yet. I found some cipher suite errata in SP 800-52r1 (confirmed by NIST) that need to be included. Essentially, there are four cipher suites that were inadvertently omitted, and the HT Bridge NIST cipher suite compliance check allows some that were not approved.

And, indeed, strict compliance with NIST SP800-52r1 does not yield optimal TLS stance.
